static/patcher-changelogs/terraform-aws-service-catalog/mgmt/openvpn-server.json

```diff
@@ -0,0 +1,1655 @@
+{
```

Comment:
We can compare this with the changelog.md https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/modules/mgmt/openvpn-server/CHANGELOG.md, and have the same result!